src/infinity-menu.js

```diff
@@ -114,7 +114,16 @@ export default class InfinityMenu extends React.Component {
 			const filteredSubFolder = node.children.length ? node.children.reduce((p, c, k) => {
 				return this.findFilted(p, c, k);
 			}, []) : [];
-			if (filteredSubFolder.length !== 0) {
+			const shouldDisplay = filteredSubFolder.reduce((prev, child) => {
```

Comment:
does this loop over all items in a folder? might be better served as a for loop or a `.some` so we don't need to loop over every item once we know we are returning true